detection/c2/unexpected-talkers-macos.sql

```diff
@@ -103,7 +103,14 @@ WHERE pos.pid IN (
     AND p0.path LIKE '/var/folders/%/T/go-build%/b001/exe/main'
   )
   AND NOT (
-    unsigned_exception = '500,6,32768,gvproxy,gvproxy'
+    unsigned_exception IN (
+      '500,6,32768,gvproxy,gvproxy',
+      '500,17,123,gvproxy,gvproxy'
+    )
     AND p0.path LIKE '/opt/homebrew/Cellar/podman/%/libexec/podman/gvproxy'
   )
+  AND NOT (
+    unsigned_exception = '500,0,0,chainlink,chainlink'
```

Comment:
Can you add a filter for `protocol = 0 && remote_port = 0`?

I believe what's happening here is that the connection is closed by the time the outer query executes.

Reply:
Added in [`4abd265` (#417)](https://github.com/chainguard-dev/osquery-defense-kit/pull/417/commits/4abd265459d70846d388c48c7028fba8daf860bf).